{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+url = "https://packages.lizard.net"
+verify_ssl = true
+name = "nens"
+
+[dev-packages]
+coverage = "*"
+nose = "*"
+"zest.releaser" = "*"
+"pep8" = "*"
+pyflakes = "*"
```

Comment:
De combi pep8 en pyflakes kan je ook door "flake8" vervangen. Dat is ook wel een aardige moderniseringsstap. (Hoeft niet gelijk, hoor).